### PR TITLE
7461: Remove blobstore

### PR DESCRIPTION
**Pull request by kat1906**
Story: [AB#7461](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7461)
Task: [AB#7644](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7644)

### CODE REVIEW

File: app/routes.test.js [hunk 1]
```diff
@@ -158,20 +166,23 @@ describe('routes', () => {
...
-            endpoint: `${config.documentApiHost}/api/v1/documents/compare-solutions.xlsx`,
-            logger,
-          });
-          getDocument.mockReset();
+    it('should call getDocumentByFileName with the correct params when the user is authorised', () => request(setUpFakeApp())
+      .get('/solutions/compare/document')
+      .then(() => {
+        expect(documentController.getDocumentByFileName.mock.calls.length).toEqual(1);
+        expect(documentController.getDocumentByFileName).toHaveBeenCalledWith({
+          res: expect.any(Object),
```
Comment (alibombaye) [inline app/routes.test.js:174]: what's this about? and does it add any value

Comment (kat1906) [reply app/routes.test.js:174]: no but without it the test will fail

Comment (kat1906) [reply app/routes.test.js:174]: also this was in the other PR too 😄

Comment (alibombaye) [reply app/routes.test.js:174]: ok fair enough 👍  - if the test needs it then lets leave it in

**Pull request merged**